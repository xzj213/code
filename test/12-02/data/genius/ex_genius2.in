8
100 120
0.492 0.230 0.425 0.238 0.315 0.376 0.343 0.433 0.339 0.290 0.280 0.293 0.331 0.277 0.289 0.504 0.342 0.371 0.414 0.430 0.454 0.402 0.468 0.433 0.262 0.244 0.505 0.266 0.287 0.317 0.233 0.405 0.373 0.248 0.243 0.233 0.437 0.451 0.414 0.308 0.461 0.226 0.469 0.227 0.329 0.302 0.335 0.357 0.323 0.379 0.278 0.330 0.253 0.374 0.469 0.473 0.391 0.305 0.386 0.310 0.330 0.251 0.347 0.244 0.478 0.317 0.279 0.322 0.437 0.447 0.331 0.501 0.350 0.241 0.474 0.331 0.261 0.365 0.381 0.393 0.434 0.431 0.280 0.482 0.330 0.362 0.414 0.395 0.295 0.458 0.497 0.269 0.316 0.304 0.409 0.224 0.327 0.505 0.227 0.315 
36 92
95 70
28 43
14 32
34 60
93 86
2 9
52 81
27 5
51 89
85 77
79 1
27 42
48 91
96 28
88 18
78 23
89 15
23 53
18 74
70 91
26 76
52 36
63 24
1 17
47 18
96 85
25 13
98 20
62 1
11 81
88 14
80 46
23 24
67 19
57 46
48 71
22 33
29 55
9 44
34 35
21 92
48 52
41 65
17 5
39 46
44 1
62 59
57 48
35 90
64 23
75 85
20 6
24 6
12 17
31 8
90 4
59 80
82 89
44 94
52 8
79 38
12 45
65 42
8 98
35 50
26 34
55 43
87 72
18 58
75 18
16 36
32 7
96 86
32 51
26 58
68 0
63 15
91 99
44 61
0 29
37 92
75 22
69 48
21 10
8 72
97 0
19 39
6 66
38 49
34 40
25 3
25 93
83 67
82 30
54 75
56 55
84 33
63 73
57 47
65 3
17 57
74 8
79 96
56 71
2 11
94 67
51 44
94 63
23 57
87 96
95 56
9 96
71 41
92 40
97 39
44 86
7 38
59 14
87 97
100 120
0.450 0.459 0.378 0.501 0.437 0.277 0.439 0.418 0.247 0.409 0.337 0.293 0.276 0.296 0.419 0.336 0.453 0.227 0.475 0.249 0.335 0.288 0.444 0.497 0.423 0.337 0.434 0.435 0.355 0.409 0.417 0.495 0.309 0.252 0.403 0.274 0.373 0.338 0.435 0.416 0.284 0.468 0.381 0.412 0.413 0.362 0.262 0.250 0.243 0.409 0.317 0.318 0.409 0.408 0.431 0.284 0.443 0.472 0.319 0.269 0.348 0.469 0.442 0.497 0.428 0.424 0.229 0.435 0.244 0.430 0.434 0.478 0.296 0.435 0.292 0.475 0.348 0.490 0.406 0.330 0.292 0.499 0.470 0.421 0.282 0.381 0.237 0.333 0.311 0.290 0.243 0.414 0.351 0.430 0.227 0.265 0.478 0.478 0.290 0.292 
41 7
87 99
31 45
47 32
40 41
12 93
89 83
65 72
5 67
95 15
82 69
18 30
0 72
76 18
35 95
83 24
54 33
55 9
33 19
81 41
47 2
70 44
34 60
24 52
29 18
81 20
85 43
94 53
48 84
66 70
70 46
42 69
4 93
62 28
47 99
85 74
4 79
50 13
30 0
89 12
82 83
47 18
18 59
49 68
90 50
36 57
32 98
48 98
50 66
88 37
16 55
19 57
48 26
31 57
61 39
94 52
93 68
17 15
28 42
34 69
57 64
33 39
78 65
79 45
20 47
50 73
8 60
88 45
94 65
98 70
40 38
95 61
97 28
66 91
91 86
23 22
97 22
67 9
55 36
10 30
92 93
43 42
14 23
99 3
77 52
37 63
7 21
18 71
42 25
1 93
36 27
75 76
56 48
56 6
60 11
58 69
32 51
72 96
87 80
93 82
38 24
33 93
17 75
9 53
89 6
63 12
62 45
93 80
2 4
70 53
37 64
42 85
21 87
62 1
84 31
56 42
81 87
27 82
7 84
57 62
100 120
0.387 0.309 0.376 0.315 0.337 0.436 0.365 0.464 0.345 0.403 0.261 0.373 0.449 0.361 0.412 0.358 0.405 0.464 0.352 0.422 0.237 0.225 0.278 0.491 0.490 0.426 0.339 0.246 0.379 0.251 0.394 0.490 0.502 0.377 0.319 0.487 0.458 0.489 0.451 0.296 0.292 0.429 0.306 0.304 0.319 0.343 0.352 0.446 0.467 0.415 0.381 0.267 0.336 0.348 0.222 0.288 0.298 0.481 0.385 0.482 0.404 0.473 0.464 0.446 0.478 0.249 0.368 0.401 0.400 0.279 0.331 0.318 0.303 0.267 0.408 0.350 0.242 0.278 0.336 0.281 0.418 0.482 0.236 0.362 0.358 0.357 0.504 0.300 0.325 0.459 0.337 0.225 0.249 0.446 0.406 0.420 0.279 0.288 0.304 0.382 
20 69
1 13
0 83
62 57
84 51
41 71
62 81
17 64
68 52
54 81
99 88
51 1
59 8
98 89
20 45
95 29
25 31
80 90
52 43
28 88
89 52
26 6
9 88
32 64
9 47
49 26
6 27
90 39
28 53
86 13
7 42
66 92
79 24
54 56
40 45
31 98
95 13
44 48
42 47
39 2
90 19
1 5
10 97
77 75
45 26
72 59
26 43
74 82
44 63
65 84
75 18
45 96
46 6
93 75
42 51
7 64
88 36
23 85
33 99
3 56
72 71
17 31
77 25
76 26
17 10
58 85
81 92
44 67
39 15
38 24
72 27
14 62
78 6
63 64
52 14
13 21
44 2
35 2
11 24
7 24
91 3
7 37
0 7
87 63
50 22
39 85
2 74
94 91
22 3
4 19
77 60
61 4
62 12
16 87
70 75
85 73
34 73
12 30
90 55
86 22
91 14
24 89
79 84
87 39
20 90
31 12
42 80
87 60
35 55
54 42
50 58
60 57
15 37
97 19
99 95
84 88
98 76
61 46
67 40
46 13
100 120
0.439 0.400 0.432 0.265 0.253 0.274 0.417 0.392 0.468 0.315 0.421 0.427 0.301 0.418 0.500 0.362 0.482 0.447 0.424 0.433 0.413 0.436 0.482 0.457 0.288 0.265 0.309 0.471 0.348 0.249 0.400 0.262 0.342 0.502 0.286 0.360 0.502 0.377 0.222 0.360 0.276 0.268 0.479 0.497 0.361 0.227 0.357 0.459 0.256 0.374 0.256 0.390 0.398 0.352 0.389 0.351 0.289 0.337 0.265 0.309 0.450 0.359 0.320 0.262 0.298 0.433 0.482 0.286 0.398 0.385 0.378 0.496 0.237 0.323 0.294 0.505 0.484 0.256 0.453 0.417 0.337 0.441 0.378 0.454 0.498 0.399 0.445 0.467 0.357 0.255 0.446 0.495 0.493 0.446 0.259 0.502 0.307 0.368 0.292 0.331 
91 4
48 39
57 68
65 30
49 0
85 86
61 0
42 20
78 60
35 33
11 30
5 44
70 28
59 39
69 15
74 78
1 25
53 36
1 62
45 39
75 0
74 91
30 81
9 14
96 25
67 61
24 23
41 33
25 90
30 98
76 24
65 54
43 24
68 98
1 35
29 32
58 99
86 48
16 75
63 6
69 31
72 15
42 69
46 67
69 91
93 65
62 83
8 31
68 14
56 6
83 92
14 50
18 80
54 44
70 57
63 55
92 34
99 23
0 39
69 79
59 92
90 28
69 45
6 31
91 38
84 25
23 18
51 77
53 23
10 7
21 18
97 99
48 47
32 93
63 58
11 27
0 82
87 51
12 70
77 19
51 4
2 84
11 7
52 74
60 26
83 3
53 64
73 25
71 16
91 37
95 22
0 88
17 71
95 46
89 96
93 40
25 13
66 32
92 94
9 97
50 28
67 66
79 40
84 48
41 79
87 73
75 32
55 31
36 49
64 1
8 82
43 11
63 12
75 19
92 47
43 76
96 1
10 73
18 76
96 28
100 120
0.227 0.403 0.500 0.456 0.414 0.231 0.227 0.388 0.365 0.332 0.309 0.444 0.268 0.252 0.250 0.337 0.430 0.470 0.249 0.265 0.488 0.465 0.434 0.305 0.317 0.388 0.466 0.324 0.280 0.463 0.258 0.445 0.292 0.472 0.472 0.225 0.370 0.423 0.272 0.469 0.329 0.500 0.416 0.503 0.362 0.238 0.319 0.432 0.387 0.465 0.356 0.412 0.437 0.477 0.294 0.378 0.339 0.333 0.423 0.453 0.387 0.380 0.386 0.224 0.370 0.455 0.429 0.331 0.257 0.401 0.250 0.363 0.306 0.435 0.248 0.445 0.466 0.433 0.474 0.377 0.287 0.261 0.492 0.288 0.240 0.503 0.400 0.258 0.454 0.256 0.320 0.296 0.420 0.317 0.330 0.446 0.268 0.494 0.225 0.468 
88 7
80 71
46 69
26 28
28 37
51 45
62 41
62 45
68 95
42 30
51 22
82 37
40 79
18 93
67 20
45 66
59 45
43 1
19 63
2 91
0 48
4 22
45 26
3 80
87 85
73 40
68 98
18 48
32 77
52 27
78 37
12 74
41 5
52 35
75 36
77 97
85 56
12 67
42 82
94 56
72 50
57 22
52 9
29 38
74 43
51 96
60 97
93 94
32 70
26 27
22 72
39 74
24 92
85 37
7 37
76 79
23 81
49 63
33 41
71 16
91 14
38 94
45 60
87 40
89 7
27 91
26 12
23 68
21 95
99 48
58 63
49 2
17 7
89 55
65 35
51 6
61 37
23 20
9 84
13 19
24 64
2 24
53 95
15 53
54 33
57 34
90 61
6 16
25 26
31 57
1 8
43 46
38 75
22 86
27 10
41 83
47 80
26 11
44 60
11 71
66 43
94 36
93 12
12 24
12 29
40 9
13 64
6 5
68 6
40 90
49 98
67 18
14 56
59 33
71 60
22 32
45 68
43 62
50 73
57 80
100 120
0.307 0.442 0.504 0.300 0.322 0.229 0.307 0.366 0.436 0.226 0.407 0.238 0.477 0.225 0.242 0.295 0.302 0.364 0.394 0.251 0.486 0.427 0.464 0.475 0.248 0.371 0.292 0.360 0.437 0.312 0.367 0.273 0.277 0.458 0.297 0.237 0.360 0.362 0.417 0.284 0.224 0.310 0.482 0.394 0.324 0.337 0.227 0.422 0.451 0.390 0.307 0.379 0.257 0.437 0.448 0.298 0.232 0.416 0.308 0.492 0.291 0.444 0.299 0.459 0.449 0.268 0.306 0.438 0.222 0.400 0.270 0.409 0.233 0.359 0.497 0.373 0.486 0.440 0.488 0.405 0.261 0.356 0.360 0.278 0.452 0.271 0.413 0.332 0.396 0.305 0.307 0.228 0.415 0.280 0.359 0.404 0.373 0.476 0.387 0.286 
33 28
96 83
93 63
17 9
42 65
43 88
14 91
78 64
15 68
66 40
17 37
46 45
31 63
73 45
74 14
65 57
21 42
55 21
96 89
56 12
13 55
49 61
24 90
58 18
84 66
95 98
27 81
33 96
70 60
27 4
69 15
28 24
90 19
72 41
69 55
57 97
48 76
43 16
93 18
37 19
63 23
39 35
67 47
0 42
86 58
10 11
15 77
69 19
43 61
73 77
1 40
93 62
27 95
2 51
55 38
41 57
84 28
11 67
16 18
58 37
73 44
0 79
2 21
60 24
5 42
24 67
48 57
10 12
37 6
98 46
85 7
6 92
5 71
32 26
99 84
8 72
50 27
46 29
75 39
66 64
83 75
13 26
29 25
3 20
96 7
29 54
36 32
74 48
7 80
56 82
23 30
50 52
3 5
57 53
22 7
72 59
37 87
23 94
5 34
19 4
10 85
83 92
29 93
6 59
54 5
52 90
38 18
55 8
75 47
1 43
78 63
73 55
56 39
24 8
19 9
96 10
60 97
48 84
2 74
87 90
100 120
0.258 0.310 0.426 0.307 0.265 0.291 0.280 0.319 0.479 0.313 0.327 0.316 0.454 0.394 0.248 0.413 0.393 0.272 0.501 0.297 0.411 0.432 0.371 0.449 0.270 0.347 0.274 0.320 0.365 0.468 0.496 0.430 0.434 0.248 0.395 0.377 0.346 0.278 0.498 0.384 0.248 0.394 0.432 0.482 0.487 0.487 0.251 0.255 0.226 0.461 0.412 0.371 0.235 0.389 0.400 0.465 0.432 0.375 0.330 0.503 0.351 0.503 0.281 0.292 0.461 0.284 0.255 0.389 0.450 0.465 0.432 0.410 0.337 0.290 0.468 0.319 0.429 0.287 0.270 0.359 0.321 0.255 0.473 0.327 0.505 0.452 0.414 0.370 0.474 0.464 0.473 0.250 0.490 0.222 0.380 0.285 0.355 0.504 0.496 0.474 
65 64
28 87
33 39
96 45
53 97
89 7
9 53
87 90
69 48
89 72
21 52
77 67
73 86
11 13
13 46
64 56
90 76
28 25
39 55
62 24
92 57
97 28
52 24
5 81
10 98
45 55
71 51
17 62
75 50
37 20
84 99
44 21
64 3
82 49
48 60
20 39
46 27
80 55
29 62
31 8
16 4
48 79
40 87
81 94
27 49
43 38
15 96
32 87
19 26
93 51
89 64
42 93
20 87
22 96
79 27
87 16
72 34
0 85
51 97
62 66
84 37
24 32
52 85
55 46
93 47
57 40
43 92
7 77
80 18
70 51
75 69
81 45
73 60
61 14
3 43
68 79
74 14
62 8
59 72
95 34
13 2
23 25
74 8
83 3
41 38
33 6
83 26
88 10
63 40
39 78
3 88
30 82
35 41
38 91
54 65
23 58
73 12
36 37
95 1
21 77
9 87
32 33
17 64
51 48
74 90
88 56
6 9
93 72
6 64
43 48
97 70
7 0
41 72
61 60
42 5
27 11
6 34
85 65
76 71
1 12
100 120
0.370 0.362 0.237 0.399 0.266 0.423 0.441 0.386 0.366 0.447 0.350 0.413 0.432 0.334 0.471 0.294 0.272 0.257 0.485 0.273 0.468 0.382 0.396 0.289 0.279 0.329 0.244 0.265 0.258 0.362 0.260 0.365 0.304 0.331 0.416 0.459 0.308 0.224 0.338 0.299 0.281 0.400 0.422 0.273 0.407 0.247 0.440 0.241 0.320 0.501 0.290 0.338 0.467 0.497 0.282 0.389 0.428 0.235 0.431 0.314 0.450 0.485 0.305 0.454 0.263 0.473 0.345 0.334 0.337 0.231 0.235 0.314 0.266 0.283 0.410 0.395 0.340 0.493 0.370 0.348 0.236 0.387 0.296 0.378 0.415 0.253 0.278 0.489 0.454 0.461 0.290 0.328 0.399 0.490 0.417 0.400 0.226 0.431 0.322 0.224 
54 39
89 34
6 87
30 96
78 59
28 24
40 96
14 18
33 47
72 43
22 30
96 79
43 18
65 73
62 66
43 94
93 60
43 65
1 8
89 20
31 16
35 14
48 17
24 36
61 54
45 88
41 30
67 37
33 88
36 60
0 49
15 24
42 66
84 65
90 35
11 74
9 36
81 2
74 0
25 40
27 36
7 10
76 77
39 98
16 57
82 68
41 94
8 94
47 90
93 66
26 43
56 74
51 82
15 19
38 28
29 4
55 57
43 5
80 16
26 51
1 67
59 26
17 82
76 54
25 27
80 92
50 90
74 19
16 68
37 4
57 76
0 32
46 92
83 31
13 83
70 29
31 89
63 93
53 43
95 27
99 2
28 86
97 48
7 18
79 23
40 91
70 3
23 12
9 85
6 21
52 70
60 99
6 19
75 0
64 33
58 44
44 35
93 71
19 69
94 89
32 85
46 36
27 47
70 66
53 72
78 39
96 39
65 12
27 88
81 57
24 5
27 62
16 65
91 33
41 99
25 10
40 12
19 36
89 46
6 88
