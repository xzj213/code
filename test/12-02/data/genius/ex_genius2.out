0
7 1 93 82 27 36 57 62 
7 7 64 17 31 98 89 24 
3 43 24 76 
5 33 59 45 62 41 
2 2 74
3 97 51 70 
3 43 72 53 
